portfolio/src/main/java/com/google/sps/servlets/DataDeleteServlet.java

```diff
@@ -45,11 +46,11 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       try {
         datastore.delete(key);
       } catch (Exception e) {
-        response.setContentType("text/html");
-        response.getWriter().println("Error with deleting comment: " + e);
+        response.setStatus(HttpServletResponse.SC_INTERNAL_SERVER_ERROR);
+        response.setContentType("application/json");
+        response.getWriter().println("{\"type\": \"SERVER\", \"message\": \"error deleting comment(s)\"}");
```

Comment:
This is theoretically fine since the serialized result only consists of constants, but please use gson.toJson() whenever crafting a result of anything other than a flat string.

Reply:
If it makes things easier, feel free to keep this as a flat string. But keep setting an appropriate return code. We can talk about how to implement structured error codes later.